Hybrid Nodes Networking Docs (pod routing, webhooks, mixed mode clusters) (#906)
awsdocs/amazon-eks-user-guide: latest/ug/nodes/hybrid-nodes-networking.adoc

*Mixed mode clusters*

Mixed mode clusters are defined as EKS clusters that have both hybrid and cloud nodes. When running a mixed mode cluster, it is recommended to run the VPC CNI on cloud nodes and either Cilium or Calico on hybrid nodes. Cilium and Calico are not supported by AWS when running on cloud nodes. It is recommended to run at least one replica of CoreDNS on cloud nodes and at least one replica of CoreDNS on hybrid nodes, see <<hybrid-nodes-add-ons#hybrid-nodes-add-ons-core, Kube-proxy and CoreDNS add-ons>> for configuration steps. See <<hybrid-nodes-add-ons, Configure add-ons for hybrid nodes>> for how to configure the webhooks used by EKS add-ons to run on cloud nodes.

> If your applications require pods running on cloud nodes to directly communicate with pods running on hybrid nodes ("east-west communication"), then your pod CIDR must be routable on your on-premises network when using the VPC CNI on cloud nodes and Cilium or Calico in overlay/tunnel mode on hybrid nodes. In this setup, the pods in the cloud get IP addresses from the VPC CNI based on the VPC subnet configuration and communicate over the VPC network. If your on-premises pod CIDR range is configured in your VPC routing table with a route to your gateway (TGW or VGW), then the traffic destined for pods on hybrid nodes will be routed to the gateway and on to your on-premises network. When the traffic reaches your on-premises network, your router must know which hybrid node owns the IP address space of the destination pod, which is why your pod CIDR must be routable on your on-premises network for this communication path to work.

Reviewer: ```suggestion
If your applications require pods running on cloud nodes to directly communicate with pods running on hybrid nodes ("east-west communication") and you are using the VPC CNI on cloud nodes and Cilium or Calico in overlay/tunnel mode on hybrid nodes, then your pod CIDR must be routable on your on-premises network. In this setup, the pods in the cloud get IP addresses from the VPC CNI based on the VPC subnet configuration and communicate over the VPC network. If your on-premises pod CIDR range is configured in your VPC routing table with a route to your gateway (TGW or VGW), then the traffic destined for pods on hybrid nodes will be routed to the gateway and on to your on-premises network. Your pod CIDR must be routable on your on-premises network, because when the traffic reaches your on-premises network, your router must know which hybrid node owns the IP address space of the destination pod.
```
